src/collectors/windows.plugin/metadata.yaml

```diff
@@ -1206,6 +1206,92 @@ modules:
               chart_type: line
               dimensions:
                 - name: uptime
+        - name: IIS W3SCV W3MP
+          description: "These metrics refer to the World Wide Web Publishing Service, the service responsible for hosting and serving web content"
+          labels:
+            - name: app
+              description: The name of the Application running.
+            - name: pid
+              description: Current PID of the process.
+          metrics:
+            - name: iis.w3scv_w3wp_active_threads
+              description: Threads actively processing requests in the worker process.
+              unit: "threads"
+              chart_type: line
+              dimensions:
+                - name: threads
+            - name: iis.w3scv_w3wp_requests_total
+              description: Maximum number of threads to which the thread pool can grow as needed.
+              unit: "requests"
+              chart_type: line
+              dimensions:
+                - name: requests
```

Comment:
If the description is correct, then metric name is wrong. This looks like a constant value. Is it? If so, I think we don't need to collect it.

Reply:
How is "Maximum number of threads to which the thread pool can grow as needed" incremented? Can you explain? What values do you get on your system?

Reply:
We currently have this data available only in our Exchange server, which is configured and maintained by our SRE team

![Screenshot_20250516_203543](https://github.com/user-attachments/assets/1b2979f2-d671-4b00-a074-a692e7ae469e)

We've shared the access permissions with you today.